docs/src/inscriptions.md

```diff
@@ -148,7 +151,12 @@ off-chain content, thus keeping inscriptions immutable and self-contained.
 
 This is accomplished by loading HTML and SVG inscriptions inside `iframes` with
 the `sandbox` attribute, as well as serving inscription content with
-`Content-Security-Policy` headers.
+`Content-Security-Policy` headers. 
+
+The iframe has a URL of `/content/<INSCRIPTION_ID>`, where `<INSCRIPTION_ID>` is 
+the ID of the rendered inscription. If an inscription has a delegate, 
+`<INSCRIPTION_ID>` is the ID of the inscription being rendered, *not* the ID of 
+the delegate.
```

Comment:
This is actually wrong for the ord explorer the /inscription page the iframe src is under /preview/ for some mime-types like html. The html preview template uses the /content/
However all this is redundant because it was stated above I would remove these changes.

Reply:
Oops, forgot about it having the preview url

I know you want this removed, but I think this part is necessary. You want it to just be that the inscription is served from `/content/:id`, and thats all we say on the matter, but just because the inscription is served from that url, doesn't mean you can use the path name to get the id (which is the whole point of this PR). Your point about `/preview/` partially demonstrates this problem. ordinals.com serves its inscriptions from `/content/:id`, but it also serves them from `/preview/:id`, which is used for the iframes in the front end. They are different urls, implying the explorer is free to choose the iframe url however they choose (provided it serves the right inscription). So, if we don't say anything about the iframes in the handbook, who's to say some explorer doesn't decide to use some other weird format for the iframe url?

The question then is what do we enforce? Do we just say it must end in `/:id`, and leave the start of the path up to the implementation?